DigitalLearningSolutions.Data.Tests/Services/NotificationPreferencesDataServiceTests.cs

```diff
@@ -92,5 +93,73 @@ public void Get_notification_preferences_for_delegate_should_return_empty_list_f
             // then
             result.Should().BeEmpty();
         }
+
+        [Test]
+        public void Update_notification_preferences_for_admin_should_update_notification_preferences_for_admin()
+        {
+            using (var transaction = new TransactionScope())
+            {
+                try
+                {
+                    // when
+                    var notificationIds = new[] { 2, 3, 4 };

```

Comment:
Do you think it might be worth structuring the test such that we filter the result by taking only accepted notification preferences and check that the ids are 2, 3 and 4? 
(no need to check the name as we're testing the GetPreferences method above)